### ISSUE TITLE
Forestry Machines no longer seem to get enough power from your conduits if any

### ISSUE THREAD

**Issue by kurokuno** [SleepyTrousers/EnderIO#161]
i have a self looping power generation on my bio fuel generation basicly i feed bio fuel into my engines it feeds back into the conduits and then end up with more then using but however as of the last update it no longer work's the machines refuse to be powered by conduits i think its because the small private engine bank can't output enough power to keep both machines running at there max input 

TLDR: i think your conduits are outputing the max into the first machine they can find reguardless of how much power it needs to run at that time
